src/test/java/integration/issue974Test.java

```diff
@@ -0,0 +1,37 @@
+package integration;
+
+import com.codeborne.selenide.SelenideConfig;
+import com.codeborne.selenide.SelenideDriver;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+
+
+class issue974Test extends ITest {
+  private SelenideDriver driver;
+
+  @BeforeEach
+  void setup() {
+    String driver1URL = getBaseUrl();
+    driver = new SelenideDriver(new SelenideConfig().baseUrl(driver1URL));
+  }
+
+  @Test
+  void frameValueSet1() {
+
+    driver.open("/issue.html");
```

Comment:
Can you re-use `page_with_frames.html` please? It has a frame with inputs.

Also, you can move `driver.open` to `@BeforeEach` method